### DISCUSSION TITLE
홍길동 - 부동산 - 기획중인 부동산 서비스 수요 조사 (예시)

### DISCUSSION THREAD

**Discussion by dinnercoding** [dinnercoding/wanted-pre-onboarding-challenge-data#91] [2023년 11월 프리온보딩 데이터 챌린지]
# 도메인을 선택한 이유
- 부동산 도메인을 선택한 이유는~

# 구체적인 문제 정의
- 부동산에서 특히, OO이 문제인데~

# 데이터 업로드
- kaggle, dacon에서 참고한 데이터입니다.

# 데이터 시각화

## 1. 부동산 가격 상승에 따른 OO 문제~~
![image](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/63527907/5ab149ff-74ef-4104-aeb3-9e81fe8a640b)

## 2. 결론
- 지금까지 부동산 OO 문제에 대해 여러 데이터를 분석해보았고, OO 결론이 나왔습니다.

**Comment by ghost** [comment]
안녕하세요
